user_data.sh

```diff
@@ -16,19 +16,22 @@ setfacl -Rdm other:0 /var/log/bastion
 # Make OpenSSH execute a custom script on logins
 echo -e "\\nForceCommand /usr/bin/bastion/shell" >> /etc/ssh/sshd_config
 
-echo "AllowTcpForwarding yes" >> /etc/ssh/sshd_config
+sed -ri 's/^#?AllowTcpForwarding .*/AllowTcpForwarding yes/' /etc/ssh/sshd_config
+
 # Block some SSH features that bastion host users could use to circumvent the solution
 awk '!/X11Forwarding/' /etc/ssh/sshd_config > temp && mv temp /etc/ssh/sshd_config
-echo "X11Forwarding no" >> /etc/ssh/sshd_config
+sed -ri 's/^#?X11Forwarding .*/X11Forwarding no/' /etc/ssh/sshd_config
 
 #hardening
 # set umask
 sed  -i "s/002/022/g" /etc/profile
 umask 022
 #disable root login
 systemctl stop sshd
-echo "PermitRootLogin no" >> /etc/ssh/ssh_config
-echo "PermitEmptyPasswords no" >> /etc/ssh/ssh_config
+/etc/ssh/sshd_config
```

Comment:
Are these two lines a typo?

Reply:
yeah sorry this wasn't ready for review yet my bad